template/dashboard.html

```diff
@@ -4,10 +4,116 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/dashboard.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-********" crossorigin="anonymous" />
+
     <title>Document</title>
-    <link rel="stylesheet" href="dashboard.css">
 </head>
 <body>
-    
+    <nav class="navbar">
+        <div class="navbar__left">
+            <div class="navbar__left-logo">
+                <img src="images/sniper.png" alt="logo">
+            </div>
+          <div class="navbar__text">AVI-HUB</div>
+        </div>
+        <div class="navbar__right">
+            <img src="images/head.jpg" alt="User Image">
+        </div>
+      </nav>
+    </nav>
+    <div class="container">
+        <div class="sidebar">
+            <div class="sidebar__card">
+                <img src="images/head.jpg" alt="logo">
+                <div class="sidebar__container">
+                  <h4><b>John Doe</b></h4> 
```

Comment:
You don't need to wrap the text with `<b>` tag, make use of css to make the font bold.